components/Button/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC, SyntheticEvent, ReactNode } from 'react';
+import { Button as ButtonUI } from '@mui/material';
+import LinkNext from 'next/link';
+import clsx from 'clsx';
+import styles from 'components/Button/button.module.scss';
```

Comment:
you can avoid full import path and use (after renaming `button.module.scss` to `index.module.scss`)
```
import styles from './index.module.scss';
````